src/test/scala/org/constellation/Fixtures.scala

```diff
@@ -47,7 +45,7 @@ object Fixtures {
   val idSet5 = Set(id1, id2, id3, id4, id5)
 
   def dummyTx(data: DAO, amt: Long = 1L) = {
-    val sendRequest = SendToAddress(id.address.address, amt)
```

Comment:
this whole PR is worth it just so I never have to type .address.address again :)

Reply:
Related question: Why's `Id` not merged into `EncodedPublicKey` anyway?

Reply:
Hahaha. I'm not sure what our convention is on using single element case classes. We could actually make SOE just a BatchSignature but I felt like there was more information there? 

We still do need an AddressMetadata class for storing additional information, so there's gonna be an addressMeta.address call somewhere in the future. 

I've seen both styles of single element case class vs. primitive String before and I'm not sure when we want to use each.

Reply:
Re: Id / EncodedPublicKey -- they both used to have different representations and serialization bugs necessitated two classes. Those were since changed and they're merged now.